packages/nodes-base/nodes/Discord/test/v2/node/channel/get.test.ts

```diff
@@ -5,10 +5,11 @@ import { getResultNodeData, setup, workflowToTests } from '@test/nodes/Helpers';
 import type { WorkflowTestData } from '@test/nodes/types';
 import { executeWorkflow } from '@test/nodes/ExecuteWorkflow';
 
+// TODO: use nock
```

Comment:
Maybe we should get into the habit of creating a story for each TODO and adding the ID on the TODO line.

Reply:
https://linear.app/n8n/issue/ENG-117